INSTALL.md

```diff
@@ -0,0 +1,164 @@
+# Easy Installation Guide for TG-Spam
+
+This guide is designed for users who want to set up TG-Spam but don't have technical experience. We'll walk through the process step by step in plain language.
+
+## What You'll Need Before Starting
+
+1. A computer running Windows, Mac, or Linux
+2. A Telegram account
+3. Admin rights in the Telegram group you want to protect
+4. About 15-20 minutes of your time
+
+## Step 1: Getting Your Bot Token from Telegram
+
+First, we need to create a bot with Telegram:
+
+1. Open Telegram and search for "@BotFather"
+2. Click "Start" to begin chatting with BotFather
+3. Type `/newbot` and send it
+4. BotFather will ask for a name for your bot. Type any name you like (e.g., "My Group's Spam Protector")
+5. Next, create a username for your bot. It must end in "bot" (e.g., "mygroupspambot" or "my_group_spam_bot")
+6. BotFather will give you a token - it looks like a long string of numbers and letters. **Save this token somewhere safe - you'll need it later!**
+
+Remember: Never share your bot token with anyone - it's like a password for your bot!
+
+## Step 2: Setting Up Docker Desktop (The Program That Runs TG-Spam)
+
+Docker Desktop is a program that helps run TG-Spam on your computer. Here's how to install it:
+
+### For Windows:
+1. Go to [Docker Desktop for Windows](https://www.docker.com/products/docker-desktop/)
+2. Click the "Download for Windows" button
+3. Once downloaded, double-click the installer
+4. Follow the installation wizard, keeping all default settings
+5. Restart your computer when asked
+
+### For Mac:
+1. Go to [Docker Desktop for Mac](https://www.docker.com/products/docker-desktop/)
+2. Click the "Download for Mac" button
+3. Once downloaded, drag Docker to your Applications folder
+4. Double-click Docker in Applications to start it
+5. Follow any prompts that appear
+
+### For Linux:
```

Comment:
Copypasting Ubuntu instructions is a bad idea, I think it will lead to discrepancies and something breaking long-term. Your call.